encoding/protobuf/inbound.go

```diff
@@ -93,6 +93,18 @@ func (o *onewayHandler) HandleOneway(ctx context.Context, transportRequest *tran
 	return o.handleOneway(ctx, request)
 }
 
+type streamHandler struct {
```

Comment:
Just an idea - should we add something to `api/transport` that is along the lines of:

```go
type StreamHandlerFunc func(ServerStream) error

func (s StreamHandlerFunc) HandleStream(serverStream ServerStream) error {
  return s(serverStream)
}
```

And do the same for Unary and Oneway?

Reply:
seems reasonable, possible to do in a follow-up since all these apis are hidden.